### DISCUSSION TITLE
Allow restoring Ark Ascended servers from internal save game backips

### DISCUSSION THREAD

**Discussion by Logan9312** [gameserverapp/Platform#678] [Suggestions]
<img width="1055" alt="image" src="https://github.com/gameserverapp/Platform/assets/87048158/9b773481-826b-4c21-91cd-811d404ef407">

In this folder: `C:\GameServerApp\containers\cakabgmsz\serverfiles\ShooterGame\Saved\SavedArks\[Map_Name]` there is a whole list of backup files, named along the lines of `TheIsland_WP_24.12.2023_19.34.27.ark`. You can restore these backups by renaming them to `TheIsland_WP.ark` and launching the server. Although this would overwrite the current file, so it might be worth renaming the current file first, then renaming the backup.

To restore a server I had to do this manually with remote desktop. I would love if GSA had a ui on the website for restoring these internal save game backups

**Comment by mvaessen** [comment]
Hi, since this is very specific to ark and fairly easy to do via ftp, is this not something I'm planning on adding.
